packages/xarc-react-router/package.json

```diff
@@ -48,7 +48,7 @@
     "eslint": "^8.16.0",
     "eslint-config-walmart": "^2.2.1",
     "eslint-plugin-filenames": "^1.3.2",
-    "eslint-plugin-jsdoc": "^39.3.2",
+    "eslint-plugin-jsdoc": "^32.0.2",
```

Comment:
is there specific reason you referred to lower version here

Reply:
At one point 39.3.2 was working fine then it started breaking out of nowhere and now it's working fine again. I'll change it back to 39.3.2.